### DISCUSSION TITLE
What is the Tab "Sync"?

### DISCUSSION THREAD

**Discussion by ravenxrz** [zellij-org/zellij#1271] [Q&A] [answered]
I don't find any information about the Tab Syn option on the official website. 

![image](https://user-images.githubusercontent.com/44746032/160092980-e61ea7b0-523b-457f-84ba-a167d9068201.png)

what does  Tab Sync use for?

**Comment by a-kenji** [comment, marked as answer]
https://zellij.dev/documentation/keybindings-actions.html#toggleactivesynctab

If you have multiple panes open, you can write into them at the same time.

**Reply by Gryzor** [reply]
Is there a new URL for this? It appears it's 404 now.

The closest I found is https://zellij.dev/documentation/cli-actions.html?highlight=sync#toggle-active-sync-tab


> Toggle between sending text input to all panes in the current tab and just to the focused pane (the default)

**Reply by tafia** [reply]
I'd say 
https://zellij.dev/documentation/faq.html?highlight=sync#how-can-i-sync-input-to-multiple-panes for people like me who accidentally enabled it.

**Reply by austinbutler** [reply]
https://zellij.dev/documentation/keybindings-possible-actions.html?highlight=sync#toggleactivesynctab now

**Reply by aarongorka** [reply]
For anyone else that stumbles on this page that can't figure out how to turn it off, it's `ctrl + t` then `s`

https://github.com/zellij-org/zellij/blob/1a2d7445201410333091fdad737db0d1add19500/zellij-utils/assets/config/default.kdl#L58
